src/cloud-api-adaptor/docs/SecureComms.md

```diff
@@ -88,9 +136,11 @@ Client->Inbound----------->Outbound->Server
 
 Inbounds and Outbounds take the form of a comma separated inbound/outbound tags such that Inbounds are formed as "InboundTag1,InboundTag2,InboundTag3,..." and Outbounds are formed as "OutboundTag1,OutboundTag2,outboundTag3,..."
 
-Each Inbound tag is structured as `Phase:Name:Port` where:
+
+Each Inbound tag is structured as `Phase:Name:Namespace:Port` or `Phase:Name:Port` where:
```

Comment:
If you use the third element of a tag for specifying a namespace, could you also update the variable name from `host` to `namespace` in `TagParse` as well?

https://github.com/confidential-containers/cloud-api-adaptor/blob/839c9d7f96af6036e4a035e0555b785fd527e88e/src/cloud-api-adaptor/pkg/securecomms/sshproxy/sshproxy.go#L204-L221

Reply:
On Outbounds we use the third element for identifying the host. 

The third element was not used for inbounds (as inbound is always open on 127.0.0.1), instead this PR suggest using it for namespace under 127.0.0.1.

Reply:
OK, I see. In that case, I think it would be helpful to add an comment in `ParseTag`.

Reply:
Added comment in ParseTag